blocks/product-details/product-details.js

```diff
@@ -26,13 +31,23 @@ import { fetchPlaceholders, setJsonLd } from '../../scripts/commerce.js';
 // Initializers
 import { IMAGES_SIZES } from '../../scripts/initializers/pdp.js';
 import '../../scripts/initializers/cart.js';
+import '../../scripts/initializers/wishlist.js';
 import { rootLink } from '../../scripts/scripts.js';
+import { readBlockConfig } from '../../scripts/aem.js';
 
 export default async function decorate(block) {
+  const {
+    'guest-wishlist-enabled': isGuestWishlistEnabled,
+  } = readBlockConfig(block);
+  block.textContent = '';
+
   // eslint-disable-next-line no-underscore-dangle
   const product = events._lastEvent?.['pdp/data']?.payload ?? null;
   const labels = await fetchPlaceholders();
 
+  // determine if Wishlist button should be enabled
+  const isAuthenticated = events._lastEvent?.['authenticated']?.payload ?? false;
```

Comment:
Exists this method in boilerplate that we can use to check if an user is authenticated or not --> **import { checkIsAuthenticated } from '../../scripts/configs.js';**

Reply:
@eliseacornejo Done!